comp/core/flare/helpers/builder.go

```diff
@@ -21,6 +21,10 @@ import (
 	"github.com/mholt/archiver/v3"
 )
 
+const (
+	filePerm = 0644
```

Comment:
I'm just curious on problems we had (if any) with the permissions that motivates this change and why we are not using it on line 31 and 248